src/odemis/acq/fastem.py

```diff
@@ -461,73 +484,20 @@ def pre_calibrate(self):
 
         NOTE: Canceling is currently not supported.
 
+        :param pre_calibrations: (list[Calibrations]) List of calibrations that should be run before the ROA
+                                 acquisition.
+        :param stage_pos: (float, float) Stage position where the calibration should be run in meter.
         """
         if not fastem_calibrations:
             raise ModuleNotFoundError("Need fastem_calibrations repository to run pre-calibrations.")
 
-        asm_config = None
+        f = align(self._scanner, self._multibeam,
+                  self._descanner, self._detector,
+                  self._stage, self._ccd,
+                  self._beamshift, self._det_rotator,
```

Comment:
It's not necessary to actually have the detector rotator defined here, you can just pass `None`. That will save a lot of code in the test cases as well. It also feels a bit strange that we would need to always pass the detector rotator to the `AcquisitionTask`, even if we do not even pre-calibrate.

I think this should work:
```
    f = align(self._scanner, self._multibeam,
              self._descanner, self._detector,
              self._stage, self._ccd,
              self._beamshift, None,
              calibrations=pre_calibrations, 
              stage_pos=stage_pos)
              
```